release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2842,8 +2842,67 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.2
+[id="ocp-4-17-2_{context}"]
+=== RHSA-2024:8229 - {product-title} {product-version}.2 image release, bug fix, and security update advisory
+
+Issued: 22 October 2024
+
+{product-title} release {product-version}.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:8229[RHSA-2024:8229] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:8232[RHBA-2024:8232] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.2 --pullspecs
+----
+
+[id="ocp-4-17-2-enhancements_{context}"]
+==== Enhancements
+
+* The Operator SDK now correctly scaffolds the Dockerfile for `kube-rbac-proxy`. Additionally, the Operator SDK can now use `-rhel9` container images. (link:https://issues.redhat.com/browse/OCPBUGS-42953[*OCPBUGS-42953*])
+
+* When you start the limited live migration method and an issue exists with network overlap, the Cluster Network Operator (CNO) can now expose network overlap metrics for the issue. This is possible because the `openshift_network_operator_live_migration_blocked` metric now includes the new `NetworkOverlap` label. (link:https://issues.redhat.com/browse/OCPBUGS-39121[*OCPBUGS-39121*])
+
+[id="ocp-4-17-2-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the approval mechanism for certificate signing requests (CSRs) failed because the node name and internal DNS entry, which is case insensitive, on a CSR did not match in terms of character case differences. With this release, an update to the approval mechanism for CSRs skips case-sensitive checks so that a CSR with a matching node name and internal DNS entry does not fail the check because of character case differences. (link:https://issues.redhat.com/browse/OCPBUGS-43312[*OCPBUGS-43312*])
```

Comment:
```suggestion
* Previously, the approval mechanism for certificate signing requests (CSRs) failed because the node name and internal DNS entry for a CSR did not match in terms of character case differences. With this release, an update to the approval mechanism for CSRs skips case-sensitive checks so that a CSR with a matching node name and internal DNS entry does not fail the check because of character case differences. (link:https://issues.redhat.com/browse/OCPBUGS-43312[*OCPBUGS-43312*])
```

Reply:
Shortened the senetence for minimalism.